### PR TITLE
Lot of changes

### PR DESCRIPTION
**Pull request by iFA88**

### CODE REVIEW

File: ico.sol [hunk 1]
```diff
@@ -379,13 +367,12 @@ contract ico is safeMath {
                 x = (value * 1e6 * USD_ETC_exchange rate / 1e4 / 1e18) * bonus percentage
                 2.700000 token = (1e18 * 1e6 * 22500 / 1e4 / 1e18) * 1.20
         */
-        reward = (value * 1e6 * icoExchangeRate / icoExchangeRateM / 1 ether) * (ICObonus() + 100) / 100;
+        reward = (value * 1e6 * icoExchangeRate / icoExchangeRateM / 1 ether) * (safeAdd(ICObonus(), 100)) / 100;
```
Comment (Dexaran) [inline ico.sol]: Computation safety: use `safeMul` instead of `*`

File: multiOwner.sol [hunk 2]
```diff
@@ -112,16 +99,25 @@ contract multiOwner is safeMath {
             return false;
         }
     }
-    /*
-        Privates
-    */
+    /* Constants */
+    function ownersForChange() public constant returns (uint256 owners) {
+        return ownerCount * doConfirmRate / 100;
```
Comment (Dexaran) [inline multiOwner.sol]: Computations safety: use `safeMul` and `safeDiv` instead of `*` and `/`

Comment (iFA88) [inline multiOwner.sol]: Please give a safeDiv function.

Comment (Dexaran) [inline multiOwner.sol]: https://github.com/Dexaran/ERC23-tokens/blob/master/SafeMath.sol#L14

Comment (iFA88) [inline multiOwner.sol]: You are joking?
```javascript
  function div(uint a, uint b) internal returns (uint) {
    // assert(b > 0); // Solidity automatically throws when dividing by 0
    uint c = a / b;
    // assert(a == b * c + a % b); // There is no case in which this doesn't hold
    return c;
}
```
What is the purpose of this function?

Comment (Dexaran) [inline multiOwner.sol]: Oh, I didn't seen that OpenZeppeling have changed the implementation of the `div` in their `safeMath` library. https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/math/SafeMath.sol#L15-L20

You're right, div is no longer needed and it should be considered deprecated.

Comment (iFA88) [inline multiOwner.sol]: Alright. I skip the safeDiv, but the safeMul will use.

Comment (Dexaran) [inline multiOwner.sol]: Yes

Comment (iFA88) [inline multiOwner.sol]: Okay im done.

**Pull request merged**